### PR TITLE
2.0 preview

### PR DESCRIPTION
**Pull request by samuelli**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -28,13 +28,27 @@
...
+  "variants": {
+    "1.x": {
+      "dependencies": {
+        "polymer": "Polymer/polymer#^1.8.0",
+        "google-apis": "GoogleWebComponents/google-apis#^1.1.7",
+        "iron-resizable-behavior": "PolymerElements/iron-resizable-behavior#^1.0.5",
+        "iron-selector": "PolymerElements/iron-selector#^1.5.2"
+      },
+      "devDependencies": {
+        "web-component-tester": "*",
```
Comment (e111077) [inline bower.json]: can you pin this to a specific version of wct?

Comment (samuelli) [inline bower.json]: Done

**Comment by samuelli** [comment]
> Please update README with slot="markers" breaking change

Done

**Comment by samuelli** [comment]
Who has merging powers? @ebidel?

**Comment by e111077** [comment]
oh I do apparently

**Comment by loopingz** [comment]
Nice thanks ! Could it be possible to merge to a 2.0-preview branch like other components ?
https://github.com/GoogleWebComponents/google-youtube/branches

**Comment by samuelli** [comment]
I can't make a new branch. @e111077?

**Comment by e111077** [comment]
Made

**Pull request merged**